kernel/src/init.cpp

```diff
@@ -111,7 +119,7 @@ extern "C" TEXT_FREE_AFTER_INIT void _start() {
 
     init_cpu();
 
-    pmm_release_bootloader_resources();
+    /// pmm_release_bootloader_resources();
```

Comment:
Apparently we are setting some memory as usable which shouldn't really be set as usable